### ISSUE TITLE
rule `exported` add `checkPublicInterface` option to check documentation in public interface definition

### ISSUE THREAD

**Issue by comdiv** [mgechev/revive#1002]
For now this code is valid:

```go
// Some - some interface
type Some interface {
     // Correct - should do all correct
     Correct()
     // not valid format of comment
     SemiCorrect()
     NonCorrect() // no documentation
}
```

suggest to add `checkPublicInterface` to `exported` rule to check documentation on interface method declaration,
so here `SemiCorrect()` has not valid comment, and `NonCorrect()` doesn't have it at all

**Comment by chavacava** [comment]
Hi @comdiv, thank you for reporting the issue and opening a PR (I'll check it in the next days)

**Comment by arxeiss** [comment]
I see the #1002 was merged, however no release was made since then. Can you please release new version so we can use this feature with GolangCI-Lint?

Thank you

**Comment by zignd** [comment]
@mgechev, @chavacava, or other collaborators. Do you have any plans for when to release a new version so that we can use this new flag?

**Comment by arxeiss** [comment]
Based on https://github.com/mgechev/revive/releases/tag/v1.4.0 it should be released. 
I guess this issue can be closed.
